### ISSUE TITLE
Does not work with Babel 6

### ISSUE THREAD

**Issue by 0xR** [ooflorent/babel-plugin-graphql#3]
Babel 6 gives the error: `Transformer is not a constructor`.

Some research shows that this is due to this plugin only supporting babel 5.

**Comment by ooflorent** [comment]
Exactly. This is a Babel 5 plugin. I haven't updated it to Babel 6 because **this project is no longer maintained**. It was created prior `graphql-js` and `react-relay`.
